docs/en/observability/apm/feature-roles.asciidoc

```diff
@@ -62,18 +82,60 @@ In general, there are three types of privileges you'll work with:
 ***********************************  ***********************************
 ////
 
-[[apm-privileges-to-publish-events]]
-=== Grant privileges and roles needed for writing events
+[float]
+[[apm-privileges-agent-central-config]]
+=== Create a _central config_ role
 
-++++
-<titleabbrev>Create a _writer_ user</titleabbrev>
-++++
+[[apm-central-config-role-note]]
+[IMPORTANT]
+====
+The privileges included in this role are *required* for all users when <<apm-agent-configuration,central configuration management>> is enabled (it is enabled by default). You do _not_ need the _Central configuration management_ role only if central configuration management has been explicitly disabled in the APM UI.
+====
 
-APM users that publish events to {es} need privileges to write to APM data streams.
+[[apm-privileges-agent-central-config-server]]
+APM Server acts as a proxy between your APM agents and the APM UI.
+The APM UI communicates any changed settings to APM Server so that your agents only need to poll the Server
+to determine which central configuration settings have changed.
+
+To grant an APM Server user with the required privileges for managing central configuration in {es} without {kib},
+assign the user the following privileges:
+
+[options="header"]
+|====
+|Type | Privilege | Purpose
+
+| Index
+|`read` on `.apm-agent-configuration` index, `allow_restricted_indices: true`
+|Allow APM Server to manage central configurations in {es}
+|====
+
+The above privileges should be sufficient for APM agent central configuration to work properly
+as long as APM Server communicates with {es} successfully.
+If it fails, it may fallback to read agent central configuration via {kib} if configured,
+which requires the following privileges:
+
+[options="header"]
+|====
+|Type | Privilege | Purpose
+
+| Spaces
+|`Read` on APM UI
+|Allow APM Server to manage central configurations via the APM UI
+|====
+
+TIP: Looking for privileges and roles needed to use central configuration from the APM UI or APM UI API?
+See <<apm-app-central-config-user,APM UI central configuration user>>.
+
+////
+***********************************  ***********************************
+***********************************  ***********************************
+////
 
 [float]
-==== General writer role
+[[apm-privileges-to-publish-events]]
+=== Create a _writer_ role
 
+APM users that publish events to {es} need privileges to write to APM data streams.
```

Comment:
Add a note that this is not needed if apm-server doesn't write to ES directly, although that is _very_ rare. Also, in those cases, the other configured ES roles will need to be passed in another section e.g. `apm-server.agent.config.elasticsearch` instead of `output.elasticsearch`.

Reply:
Moving all related discussion to https://github.com/elastic/observability-docs/pull/4193#discussion_r1747383918.